config.md

```diff
@@ -104,10 +104,10 @@
 |credentials|CORS setting to control whether a browser allows credentials to be sent to this API|`boolean`|`true`
 |debug|Whether debug is enabled for the CORS implementation|`boolean`|`false`
 |enabled|Whether CORS is enabled|`boolean`|`true`
-|headers|CORS setting to control the allowed headers|`string`|`[*]`
+|headers|CORS setting to control the allowed headers|`[]string`|`[*]`
```

Comment:
This one is here: https://github.com/hyperledger/firefly-common/blob/5a2539a3727b6eeecee07da0297198f7bcbd2bed/pkg/i18n/en_base_config_descriptions.go#L104

Reply:
https://github.com/hyperledger/firefly-common/pull/36